OnShift/SendBird-JavaScript
# PL 73 no more login

# Things Done
- remove login screen from code
  - the consumer of the widget has to provide credential on start up via `startWithConnect`
- remove login screen styling
- tweak notification code
  - better, more informative notifications depending on the type of message
- minor SASS tweaks
  - cache another image locally
  - never see 'connecting...' image

# How To Test
- try to see the login screen
- try to get the widget in a state where it expects user input in order to function
- to test notifications
  - spin up another instance of the widget in another window
    - I've been using an incognito window to achieve this
  - start passing messages between the two logged in users
    - to see the notifications, make sure the non-incognito user does not have the chat widget open
